java/google/registry/flows/domain/DomainTransferUtils.java

```diff
@@ -104,22 +103,29 @@ public static TransferData createPendingTransferData(
         .add(createTransferBillingEvent(
             automaticTransferTime,
             historyEntry,
-            targetId,
+            existingDomain.getRepoId(),
```

Comment:
This change was wrong -- targetId is the fullyQualifiedDomainName, not the repoId.  I fixed it for you.